html/pfappserver/lib/pfappserver/Form/Config/Roles.pm

```diff
@@ -104,6 +108,19 @@ has_field 'inherit_web_auth_url' => (
     default => 'disabled',
 );
 
+sub _validate_acl {
```

Comment:
This function should be in a util module somewhere in `lib/` so it can be reused elsewhere